src/pelz-service/main.c

```diff
@@ -95,6 +95,8 @@ int main(int argc, char **argv)
     pelz_log(LOG_ERR, "Key Table Init Failure");
     return (1);
   }
+
+  kmyth_unsealed_data_table_initialize(eid, &ret);
```

Comment:
With both this and ```private_pkey_init``` there's a bunch of cleanup that doesn't happen on errors, it only happens if everything succeeds and you get to the end of main.